com.unity.netcode.gameobjects/Tests/Runtime/NetworkObject/NetworkObjectPropertyTests.cs

```diff
@@ -0,0 +1,63 @@
+using NUnit.Framework;
+using System.Collections;
+using Unity.Netcode.TestHelpers.Runtime;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Unity.Netcode.RuntimeTests
+{
+    /// <summary>
+    /// Tests properties of NetworkObject for proper functinality.
+    /// </summary>
+    public class NetworkObjectPropertyTests : NetcodeIntegrationTest
+    {
+        protected override int NumberOfClients => 1;
+
+        private NetworkPrefab m_TestPrefab;
+
+        protected override void OnServerAndClientsCreated()
+        {
+            // create prefab
+            var gameObject = new GameObject("TestObject");
```

Comment:
Just FYI:
`CreateNetworkObjectPrefab` handles all of this for you (and then some).